test/functional/wallet_descriptor.py

```
@@ -4,6 +4,15 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test descriptor wallet function."""
 
+try:
+    import sqlite3
+except ImportError:
+    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ImportError)`

(SIM105)